.github/workflows/auto_request_review.yml

```diff
@@ -3,11 +3,14 @@ name: Auto Request Review
 on:
   pull_request:
     types: [opened, ready_for_review, reopened]
-
+   
 jobs:
   auto-request-review:
     # Don't run on forked repos
     if: ${{ !github.event.pull_request.head.repo.fork }}
+    permissions:
+     contents: read
+      pull-requests: write
```

Comment:
Is this indentation correct?

Reply:
Fixed, but hopefully CI would have caught that :P